packages/flex-plugin-scripts/src/scripts/deploy.ts

```diff
@@ -187,7 +196,7 @@ export const _doDeploy = async (nextVersion: string, options: Options): Promise<
           logger.newline();
           logger.warning('Plugin already exists and the flag --overwrite is going to overwrite this plugin.');
         }
-      } else {
+      } else if (env.isCI()) {
```

Comment:
What would happen if I run this script manually without going through the CLI, and I have a duplicate? This error would never be thrown no?

Reply:
it wouldn't be thrown, which is what we want, right? since we only want to block if it's on CI

Reply:
I meant if I use this the old way, without the CLI, and do `npm run deploy` (and have `flex-plugin deploy` in my scripts), then even locally, this wouldn't throw this error because `env.isCI()` is false but it should still throw the error.

Reply:
from what ruma said it sounded like we don't ever want to throw this error unless it's CI, but i'll clarify the `npm run deploy` use case and make sure